spec/request/api/registration/registrations_spec.rb

```diff
@@ -0,0 +1,122 @@
+RSpec.describe "POST /api/sign_up", type: :request do
+  let(:headers) { { HTTP_ACCEPT: "application/json" } }
+
+  describe "with valid credentials" do
+    before do
+      post "/api/auth",
+           params: {
+             email: "client@mail.com",
+             password: "password",
+             password_confirmation: "password",
+             role: "client",
+             company_name: "company name",
+             company_url: "http://develup.com",
+           },
+           headers: headers
+    end
+
+    it "is expected to respond with ok status" do
+      expect(response).to have_http_status :ok
+    end
+
+    it "returns a succesfully message" do
+      expect(response_json["status"]).to eq "success"
+    end
+  end
```

Comment:
is expected

Reply:
No. We use is expected for the unit test